spec/unit/block_score/actions/delete_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'faker'
+
+RSpec.describe BlockScore::Actions::Delete do
+  describe '#delete' do
+    subject(:candidate) { create(:candidate) }
+    before { candidate.delete }
+
+    it { is_expected.not_to be_persisted }
+    it { expect(candidate.deleted).to be true }
```

Comment:
This isn't something for this PR, but I wonder if the method should be `#deleted?` rather than `#deleted`? The boolean method could be added, and then `#deleted` could use it and/or optionally warn that it is a deprecated method.